### PR TITLE
Feat/autocomplete component

### PR DESCRIPTION
**Pull request by NoyKerenHLS**
create autocomplete component

### CODE REVIEW

File: src/components/autocomplete/Autocomplete.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  Box,
+} from "@mui/material";
+import Icon from "../Icon/Icon";
+import search from "../../assets/autocomplete/search.svg";
+import XIcon from "../../assets/autocomplete/XIcon.svg";
+import { headLineStyle, listItemStyle, searchBarAutocomplete } from "./styles";
+import { useState } from "react";
+
+interface Props {
+  options: string[];
```
Comment (AyalWein) [inline src/components/autocomplete/Autocomplete.tsx:20]: objects

File: src/components/autocomplete/Autocomplete.tsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  icon?: string;
+  handleSearch: (value: string) => void;
+}
+
+const Autocomplete = ({ options, icon = search, handleSearch }: Props) => {
+  const headLine = "Recent Searches";
+  const style = searchBarAutocomplete;
+
+  const [inputValue, setInputValue] = useState("");
```
Comment (AyalWein) [inline src/components/autocomplete/Autocomplete.tsx]: typescript generics

File: src/components/autocomplete/Autocomplete.tsx [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+interface Props {
+  options: string[];
+  icon?: string;
+  handleSearch: (value: string) => void;
+}
+
+const Autocomplete = ({ options, icon = search, handleSearch }: Props) => {
+  const headLine = "Recent Searches";
+  const style = searchBarAutocomplete;
```
Comment (AyalWein) [inline src/components/autocomplete/Autocomplete.tsx]: use directly

**Review by AyalWein — CHANGES_REQUESTED** [review]

File: src/components/autocomplete/Autocomplete.tsx (hunk 1 above)
Comment (NoyKerenHLS) [reply src/components/autocomplete/Autocomplete.tsx:20]: Problematic, will happen later

**Review by AyalWein — APPROVED** [review]

**Pull request merged**